src/EventTickets/resources/admin/components/TicketTypeFormModal/parseValueFromLocale.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * @unreleased
+ */
+export default function parseValueFromLocale(amount: string): string {
+    if (!amount) {
+        return amount;
+    }
+
+    const numberFormat = new Intl.NumberFormat(window.navigator.language);
+    const parts = numberFormat.formatToParts(1234.56);
```

Comment:
Is `1234.56` some magic number? Or was this supposed to be the `amount` variable in its place?